jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/component/Dumpable.java

```diff
@@ -37,7 +40,7 @@
 @ManagedObject("Dumpable Object")
 public interface Dumpable
 {
-    String KEY = "key: +- bean, += managed, +~ unmanaged, +? auto, +: iterable, +] array, +@ map, +> undefined\n";
+    String LEGEND = "key: +- bean, += managed, +~ unmanaged, +? auto, +: iterable, +] array, +} map, +> pojo; @ visited\n";
```

Comment:
```suggestion
    String LEGEND = "legend: +- bean, += managed, +~ unmanaged, +? auto, +: iterable, +] array, +} map, +> pojo; @ visited\n";
```

Change also `ServerTest.testDump()` to assert that it contains the LEGEND constant.